packages/__docs__/src/Icons/index.tsx

```diff
@@ -112,24 +106,24 @@ class Icons extends Component {
     return glyphs
   }
 
-  handleSearchChange = (e) => {
-    const query = e.target.value
+  handleSearchChange = (e: SyntheticEvent) => {
+    const query = (e.target as HTMLInputElement).value
 
     clearTimeout(this.searchTimeout)
     this.searchTimeout = setTimeout(() => {
       this.setState({ query })
     }, 500)
   }
 
-  handleFormatChange = (e, o) => {
+  handleFormatChange = (o: Record<string, any>) => {
```

Comment:
As I mentioned before, the order of the parameters count, so you cannot remove `e` as a first parameter, it will break the code. If it is not used, mark it as unused:
`handleFormatChange = (_e: SyntheticEvent, o: Record<string, any>) => { ...  }`